### PR TITLE
[docs] Update 'Global configuration' article

### PR DESCRIPTION
**Pull request by Zhbert**
## Description

The "Global configuration" article reviewing and fixing.

## Changelog entries

```changes
module: docs
type: fix
description: "Review and fix the 'Global configuration' article."
impact_level: low
```

### CODE REVIEW

**Comment by deckhouse-BOaTswain** [comment]
Run workflow "deploy/web/test"...

**Comment by deckhouse-BOaTswain** [comment]
Run workflow "deploy/web/stage"...

**Comment by deckhouse-BOaTswain** [comment]
Run workflow "deploy/web/stage"...

**Comment by deckhouse-BOaTswain** [comment]
Running workflow "deploy/web/stage"...

**Comment by z9r5** [comment]
@shvgn Please review Ru-version of changes in general. Ready to discuss. Thnks

**Comment by deckhouse-BOaTswain** [comment]
Running workflow "deploy/web/stage"...

File: docs/documentation/pages/DECKHOUSE_CONFIGURE_RU.md [hunk 1]
```diff
@@ -4,51 +4,83 @@ permalink: ru/
...
+Пример ConfigMap `deckhouse`:
 ```yaml
 apiVersion: v1
 metadata:
   name: deckhouse
   namespace: d8-system
 data:
   global: |          # Вертикальная черта.
-    # Тут кусок YAML-файла с глобальными настройками.
+    # глобальные настройки в формате YAML.
```
Comment (Zhbert) [inline docs/documentation/pages/DECKHOUSE_CONFIGURE_RU.md:33]:
```suggestion
    # Глобальные настройки в формате YAML.
```

File: docs/documentation/pages/DECKHOUSE_CONFIGURE_RU.md [hunk 2]
```diff
@@ -4,51 +4,83 @@ permalink: ru/
...
+* Символ `|` (вертикальная черта) обязательно должен быть указан, т.к. передаваемое значение — многострочная строка (multi-line string), а не объект;
+* Наименование модулей пишется в стиле *camelCase*, при котором несколько слов пишутся слитно без пробелов, при этом каждое слово внутри фразы пишется с прописной буквы;
+
+Чтобы изменить конфигурацию Deckhouse отредактируйте ConfigMap `deckhouse`, например следующим способом:
+```shell
+kubectl -n d8-system edit cm/deckhouse
 ```
 
-## Включение и отключение модуля
+После сохранения конфигурации Deckhouse, изменения применяются автоматически.
```
Comment (Zhbert) [inline docs/documentation/pages/DECKHOUSE_CONFIGURE_RU.md:53]:
```suggestion
После сохранения конфигурации Deckhouse изменения применяются автоматически. 
```

File: docs/documentation/pages/DECKHOUSE_CONFIGURE_RU.md [hunk 3]
```diff
@@ -4,51 +4,83 @@ permalink: ru/
...
 
-В зависимости от используемого [варианта поставки](./modules/020-deckhouse/configuration.html) модули могут быть включены или выключены по умолчанию.
+> Deckhouse использует проект [addon-operator](https://github.com/flant/addon-operator/) при работе с [модулями](https://github.com/flant/addon-operator/blob/main/MODULES.md), [хуками модулей](https://github.com/flant/addon-operator/blob/main/HOOKS.md), [параметрами модулей](https://github.com/flant/addon-operator/blob/main/VALUES.md). Если вы хотите глубже понять архитектуру Deckhouse, познакомьтесь с проектом `addon-operator`. Будем признательны, если поставите проекту *звезду*.     
 
-Для включения или отключения модуля необходимо добавить в ConfigMap `deckhouse` параметр `<moduleName>Enabled`, который может принимать одно из двух значений: `"true"` или `"false"`, где `<moduleName>` — название модуля в camelCase.
+Deckhouse работает только с включёнными модулями. В зависимости от используемого [варианта поставки](./modules/020-deckhouse/configuration.html#parameters-bundle) модули могут быть включены или выключены по умолчанию. Читайте подробнее, про явное [включение или отключение модуля](#включение-и-отключение-модуля).
```
Comment (Zhbert) [inline docs/documentation/pages/DECKHOUSE_CONFIGURE_RU.md:59]:
```suggestion
Deckhouse работает только с включёнными модулями. В зависимости от используемого [варианта поставки](./modules/020-deckhouse/configuration.html#parameters-bundle) модули могут быть включены или выключены по умолчанию. Читайте подробнее про явное [включение или отключение модуля](#включение-и-отключение-модуля).
```

**Comment by deckhouse-BOaTswain** [comment]
Running workflow "deploy/web/stage"...

**Comment by deckhouse-BOaTswain** [comment]
Running workflow "deploy/web/stage"...

File: docs/documentation/pages/DECKHOUSE_CONFIGURE2_RU.md [hunk 4]
```diff
@@ -0,0 +1,132 @@
+---
+title: "Как настроить?"
+permalink: ru/ver2.html
+lang: ru
+---
+
+Deckhouse состоит из оператора Deckhouse и модулей. Модуль — набор helm-чартов, хуков, файлов и правил сборки компонентов модуля (компонентов Deckhouse).
+
+> При работе с модулями Deckhouse использует проект [addon-operator](https://github.com/flant/addon-operator/). Ознакомьтесь с его документацией, если хотите понять как Deckhouse работает с [модулями](https://github.com/flant/addon-operator/blob/main/MODULES.md), [хуками модулей](https://github.com/flant/addon-operator/blob/main/HOOKS.md), [параметрами модулей](https://github.com/flant/addon-operator/blob/main/VALUES.md). Будем признательны, если поставите проекту *звезду*.
```
Comment (Zhbert) [inline docs/documentation/pages/DECKHOUSE_CONFIGURE2_RU.md:9]:
```suggestion
> При работе с модулями Deckhouse использует проект [addon-operator](https://github.com/flant/addon-operator/). Ознакомьтесь с его документацией, если хотите понять как Deckhouse работает с [модулями](https://github.com/flant/addon-operator/blob/main/MODULES.md), [хуками модулей](https://github.com/flant/addon-operator/blob/main/HOOKS.md) и [параметрами модулей](https://github.com/flant/addon-operator/blob/main/VALUES.md). Будем признательны, если поставите проекту *звезду*.
```

File: docs/documentation/pages/DECKHOUSE_CONFIGURE2_RU.md [hunk 5]
```diff
@@ -0,0 +1,132 @@
...
+      - 10.2.7.80
+      zone: directory.company.my
+    upstreamNameservers:
+    - 10.2.100.55
+    - 10.2.200.55
+  # Отключение модуля dashboard.
+  dashboardEnabled: "false"   
+```
+
+Чтобы изменить конфигурацию Deckhouse отредактируйте ConfigMap `deckhouse`, например следующим способом:
```
Comment (Zhbert) [inline docs/documentation/pages/DECKHOUSE_CONFIGURE2_RU.md:51]:
```suggestion
Чтобы изменить конфигурацию Deckhouse отредактируйте ConfigMap `deckhouse`, например, следующим способом:
```

File: docs/documentation/pages/DECKHOUSE_CONFIGURE2_RU.md [hunk 6]
```diff
@@ -0,0 +1,132 @@
...
+</tbody>
+</table>
+
+Выключить модули, включенные по умолчанию в используемом наборе модулей (и наоборот), вы можете в [конфигурации Deckhouse](#конфигурация-deckhouse).
+
+## Выделение узлов под определенный вид нагрузки
+
+Если в параметрах модуля не указаны явные значения `nodeSelector/tolerations`, то для всех модулей используется следующая стратегия:
+
+1. Если параметр `nodeSelector` модуля не указан, то Deckhouse попытается вычислить `nodeSelector` автоматически. В этом случае, если в кластере присутствуют узлы с [лейблами из списка или лейблами определенного формата](#особенности-автоматики-зависящие-от-типа-модуля), то Deckhouse укажет их в качестве `nodeSelector` ресурсам модуля;
```
Comment (Zhbert) [inline docs/documentation/pages/DECKHOUSE_CONFIGURE2_RU.md:95]:
```suggestion
1. Если параметр `nodeSelector` модуля не указан, то Deckhouse попытается вычислить `nodeSelector` автоматически. В этом случае, если в кластере присутствуют узлы с [лейблами из списка или лейблами определенного формата](#особенности-автоматики-зависящие-от-типа-модуля), Deckhouse укажет их в качестве `nodeSelector` ресурсам модуля;
```

I'm not sure about this edit, but it seems to me that the preposition is not needed here.

File: docs/documentation/pages/DECKHOUSE_CONFIGURE2_RU.md [hunk 7]
```diff
@@ -0,0 +1,132 @@
...
+1. Если параметр `tolerations` модуля не указан, то Pod'ам модуля автоматически устанавливаются все возможные toleration'ы ([подробнее](#особенности-автоматики-зависящие-от-типа-модуля));
+1. Отключить автоматическое вычисление параметров `nodeSelector` или `tolerations` можно указав значение `false`.
```
Comment (Zhbert) [inline docs/documentation/pages/DECKHOUSE_CONFIGURE2_RU.md:97]:
```suggestion
1. Отключить автоматическое вычисление параметров `nodeSelector` или `tolerations` можно, указав значение `false`.
```

File: docs/documentation/pages/DECKHOUSE_CONFIGURE_RU.md [hunk 8]
```diff
@@ -4,94 +4,160 @@ permalink: ru/
...
+  monitoringPing: |
+    externalTargets:
+    - host: 8.8.8.8
+  # Отключение модуля dashboard.
+  dashboardEnabled: "false"   
+```
+
+Обратите внимание на несколько важных нюансов в конфигурации:
+* Символ `|` (вертикальная черта) обязательно должен быть указан, т.к. передаваемое значение — многострочная строка (multi-line string), а не объект;
+* Наименование модулей пишется в стиле *camelCase*, при котором несколько слов пишутся слитно без пробелов, при этом каждое слово внутри фразы пишется с прописной буквы;
```
Comment (Zhbert) [inline docs/documentation/pages/DECKHOUSE_CONFIGURE_RU.md:46]:
```suggestion
* Наименование модулей пишется в стиле *camelCase*, при котором несколько слов пишутся слитно без пробелов, при этом каждое слово внутри фразы пишется с прописной буквы.
```

File: docs/documentation/pages/DECKHOUSE_CONFIGURE_RU.md [hunk 9]
```diff
@@ -4,94 +4,160 @@ permalink: ru/
...
+
+Чтобы изменить конфигурацию Deckhouse отредактируйте ConfigMap `deckhouse`, например следующим способом:
```
Comment (Zhbert) [inline docs/documentation/pages/DECKHOUSE_CONFIGURE_RU.md:48]:
```suggestion
Чтобы изменить конфигурацию Deckhouse отредактируйте ConfigMap `deckhouse`, например, следующим способом:
```

File: docs/documentation/pages/DECKHOUSE_CONFIGURE_RU.md [hunk 10]
```diff
@@ -4,94 +4,160 @@ permalink: ru/
...
 ```
 
-## Включение и отключение модуля
+После сохранения конфигурации Deckhouse изменения применяются автоматически. 
 
-Deckhouse может установить только включённые [модули](https://github.com/flant/addon-operator/blob/master/MODULES.md). Подробнее про определение состояния модуля можно почитать [в документации](https://github.com/flant/addon-operator/blob/master/LIFECYCLE.md#modules-discovery).
+### Настройка модуля
 
-В зависимости от используемого [варианта поставки](./modules/020-deckhouse/configuration.html) модули могут быть включены или выключены по умолчанию.
+> При работе с модулями Deckhouse использует проект [addon-operator](https://github.com/flant/addon-operator/). Ознакомьтесь с его документацией, если хотите понять как Deckhouse работает с [модулями](https://github.com/flant/addon-operator/blob/main/MODULES.md), [хуками модулей](https://github.com/flant/addon-operator/blob/main/HOOKS.md), [параметрами модулей](https://github.com/flant/addon-operator/blob/main/VALUES.md). Будем признательны, если поставите проекту *звезду*.
```
Comment (Zhbert) [inline docs/documentation/pages/DECKHOUSE_CONFIGURE_RU.md:57]:
```suggestion
> При работе с модулями Deckhouse использует проект [addon-operator](https://github.com/flant/addon-operator/). Ознакомьтесь с его документацией, если хотите понять как Deckhouse работает с [модулями](https://github.com/flant/addon-operator/blob/main/MODULES.md), [хуками модулей](https://github.com/flant/addon-operator/blob/main/HOOKS.md) и [параметрами модулей](https://github.com/flant/addon-operator/blob/main/VALUES.md). Будем признательны, если поставите проекту *звезду*.
```

File: docs/documentation/pages/DECKHOUSE_CONFIGURE_RU.md [hunk 11]
```diff
@@ -4,94 +4,160 @@ permalink: ru/
...
+</table>
+
 ## Выделение узлов под определенный вид нагрузки
 
-Для всех модулей принята единая стратегия:
+Если в параметрах модуля не указаны явные значения `nodeSelector/tolerations`, то для всех модулей используется следующая стратегия:
 
-1. Если параметр `nodeSelector` модуля не указан, то Deckhouse попытается вычислить `nodeSelector` автоматически. В этом случае, если в кластере присутствуют узлы с лейблами из определенного списка или лейблами определенного формата (подробнее — ниже, под спойлером), то Deckhouse укажет их в качестве `nodeSelector` ресурсам модуля;
-1. Если параметр `tolerations` модуля не указан, то Pod'ам модуля автоматически устанавливаются все возможные toleration'ы (подробнее — ниже, под спойлером);
+1. Если параметр `nodeSelector` модуля не указан, то Deckhouse попытается вычислить `nodeSelector` автоматически. В этом случае, если в кластере присутствуют узлы с [лейблами из списка или лейблами определенного формата](#особенности-автоматики-зависящие-от-типа-модуля), то Deckhouse укажет их в качестве `nodeSelector` ресурсам модуля;
```
Comment (Zhbert) [inline docs/documentation/pages/DECKHOUSE_CONFIGURE_RU.md:126]:
```suggestion
1. Если параметр `nodeSelector` модуля не указан, то Deckhouse попытается вычислить `nodeSelector` автоматически. В этом случае, если в кластере присутствуют узлы с [лейблами из списка или лейблами определенного формата](#особенности-автоматики-зависящие-от-типа-модуля), Deckhouse укажет их в качестве `nodeSelector` ресурсам модуля;
```
I'm not sure about this edit, but it seems to me that the preposition is not needed here.

File: docs/documentation/pages/DECKHOUSE_CONFIGURE_RU.md [hunk 12]
```diff
@@ -4,94 +4,160 @@ permalink: ru/
...
+1. Если параметр `tolerations` модуля не указан, то Pod'ам модуля автоматически устанавливаются все возможные toleration'ы ([подробнее](#особенности-автоматики-зависящие-от-типа-модуля));
 1. Отключить автоматическое вычисление параметров `nodeSelector` или `tolerations` можно указав значение `false`.
```
Comment (Zhbert) [inline docs/documentation/pages/DECKHOUSE_CONFIGURE_RU.md:128]:
```suggestion
1. Отключить автоматическое вычисление параметров `nodeSelector` или `tolerations` можно указав, значение `false`.
```

File: global-hooks/openapi/doc-ru-config-values.yaml [hunk 13]
```diff
@@ -2,93 +2,96 @@ type: object
...
+        * Если значение не указано, то используется автоматически определяемый `cluster.defaultStorageClass`. Если `cluster.defaultStorageClass` не определен, то используется `emptyDir`.
         * Этот параметр имеет смысл использовать только в исключительных ситуациях.
   highAvailability:
     description: |
-      Глобальный включатель режима отказоустойчивости для модулей, которые это поддерживают.
-      
-      По умолчанию не определён и решение принимается на основе autodiscovery-параметра `global.discovery.clusterControlPlaneIsHighlyAvailable`.
+      Параметр глобального включения *режима отказоустойчивости* для модулей, которые его поддерживают.
+
+      Если значение не указано, то используется значение autodiscovery-параметра `global.discovery.clusterControlPlaneIsHighlyAvailable`.
```
Comment (Zhbert) [inline global-hooks/openapi/doc-ru-config-values.yaml:12]: `значение autodiscovery-параметра` — maybe replace it with something more Russian?

File: global-hooks/openapi/doc-ru-config-values.yaml [hunk 14]
```diff
@@ -2,93 +2,96 @@ type: object
...
+              Количество ресурсов (CPU и памяти), выделяемых для компонентов Deckhouse работающих на каждом master-узле (в том числе `control plane`, если `control plane` управляется Deckhouse).
+
+              **Внимание!** В случае managed-кластера Deckhouse не управляет компонентами `control plane`, поэтому все указанные в параметре `masterNode` ресурсы отдаются компонентам Deckhouse.
             properties:
               cpu:
                 description: |
-                  Сумма CPU, выделяемая для системных компонентов на master-узлах сверх `everyNode`.
-                    * Для кластера, управляемого Deckhouse, значение по умолчанию определяется автоматически: `.status.allocatable.cpu` минимального master-узла (но не более 4 ядер) минус `everyNode`.
-                    * Для managed-кластера значение по умолчанию — 1 ядро минус `everyNode`.
+                  Количество CPU, выделяемое на каждом master-узле **дополнительно** к указанному в параметре `everyNode.cpu`.
```
Comment (Zhbert) [inline global-hooks/openapi/doc-ru-config-values.yaml:90]:
```suggestion
                  Количество CPU, выделяемое на каждом master-узле **дополнительно** к указанному в параметре `everyNode.cpu`:
```

File: global-hooks/openapi/doc-ru-config-values.yaml [hunk 15]
```diff
@@ -2,93 +2,96 @@ type: object
...
+                    * Для кластера, управляемого Deckhouse, значение по умолчанию определяется автоматически по формуле: `.status.allocatable.cpu` минимального master-узла (но не более `4` (ядер CPU)) минус количество CPU, указанное в параметре `everyNode.cpu`.
```
Comment (Zhbert) [inline global-hooks/openapi/doc-ru-config-values.yaml:91]:
```suggestion
                    * Для кластера, управляемого Deckhouse, значение по умолчанию определяется автоматически по формуле: `.status.allocatable.cpu` минимального master-узла (но не более `4` (ядер CPU)) минус количество CPU, указанное в параметре `everyNode.cpu`;
 ```

File: global-hooks/openapi/doc-ru-config-values.yaml [hunk 16]
```diff
@@ -2,93 +2,96 @@ type: object
...
+                    * Для managed-кластера значение по умолчанию: `1` (одно ядро CPU) минус количество CPU, указанное в параметре `everyNode.cpu`.
               memory:
                 description: |
-                  Суммарный объем памяти, выделяемый для системных компонентов на master-узлах сверх `everyNode`.
-                    * Для кластера, управляемого Deckhouse, значение по умолчанию определяется автоматически: `.status.allocatable.memory` минимального master-узла (но не более 8 Гб) минус `everyNode`.
-                    * Для managed-кластера значение по умолчанию — 1 Гб минус `everyNode`.
+                  Объем памяти, выделяемый на каждом master-узле, **дополнительно** к указанному в параметре `everyNode.memory`.
```
Comment (Zhbert) [inline global-hooks/openapi/doc-ru-config-values.yaml:95]:
```suggestion
                  Объем памяти, выделяемый на каждом master-узле, **дополнительно** к указанному в параметре `everyNode.memory`:
```

File: global-hooks/openapi/doc-ru-config-values.yaml [hunk 17]
```diff
@@ -2,93 +2,96 @@ type: object
...
+                    * Для кластера, управляемого Deckhouse, значение по умолчанию определяется автоматически по формуле: `.status.allocatable.memory` минимального master-узла (но не более `8Gi`) минус значение, указанное в параметре `everyNode.memory`.
```
Comment (Zhbert) [inline global-hooks/openapi/doc-ru-config-values.yaml:96]:
```suggestion
                    * Для кластера, управляемого Deckhouse, значение по умолчанию определяется автоматически по формуле: `.status.allocatable.memory` минимального master-узла (но не более `8Gi`) минус значение, указанное в параметре `everyNode.memory`;
 ```

**Review by Zhbert — COMMENTED** [review]
@z9r5 Please ckeck suggestions.

File: docs/documentation/pages/DECKHOUSE_CONFIGURE2_RU.md (hunk 4 above)
Comment (z9r5) [reply docs/documentation/pages/DECKHOUSE_CONFIGURE2_RU.md]: Done

File: global-hooks/openapi/doc-ru-config-values.yaml (hunk 13 above)
Comment (z9r5) [reply global-hooks/openapi/doc-ru-config-values.yaml]: Fixed

**Comment by deckhouse-BOaTswain** [comment]
Running workflow "deploy/web/stage"...

**Comment by deckhouse-BOaTswain** [comment]
Running workflow "deploy/web/test"...

**Pull request merged**